jobs/gtfs-rt-parser/gtfs_rt_parser.py

```diff
@@ -200,32 +212,42 @@ def main(
     typer.secho(f"Parsing {file_type} from {glob}", fg=typer.colors.MAGENTA)
 
     # fetch files ----
-    feed_files = identify_files(glob=glob, rt_file_type=file_type, progress=progress)
+    files = identify_files(glob=glob, rt_file_type=file_type, progress=progress)
+
+    feed_hours = defaultdict(list)
+
+    for file in files:
+        feed_hours[file.hive_path(dst_bucket)].append(file)
+
+    feed_hours = [
+        RTAggregation(
+            hive_path=hive_path,
+            source_files=files,
+        )
+        for hive_path, files in feed_hours.items()
+    ]
 
     if limit:
         structlog.get_logger().warn(f"limit of {limit} feeds was set")
```

Comment:
Should this say "hours" rather than "feeds" now?

Reply:
it's still feeds, it's just that N is now feeds rather than all records (the log was inaccurate before tbh)

i.e. each `feed_hours` entry is one hour of one feed